dracut-iguana/iguana/iguana.sh

```diff
@@ -103,11 +98,10 @@ EOF
 # - directory with results
 
 IGUANA_BUILDIN_CONTROL="/etc/iguana/control.yaml"
-IGUANA_CMDLINE_EXTRA="--newroot=${NEWROOT} ${IGUANA_DEBUG:+--debug --log-level=debug}"
+IGUANA_CMDLINE_EXTRA="--newroot='${NEWROOT}' ${IGUANA_DEBUG:+--debug --log-level=debug}"
 
 if [ -n "$IGUANA_CONTROL_URL" ]; then
-  curl --insecure -o control_url.yaml -L -- "$IGUANA_CONTROL_URL"
-  if [ $? -ne 0 ]; then
+  if ! curl --insecure -o control_url.yaml -L -- "$IGUANA_CONTROL_URL"; then
```

Comment:
We should make the `--insecure` argument optional or, at least, document that we are using it.

Reply:
I will address this in another PR.
First I need to find some way how people can add custom certificates.
Once we start with signing iguana initrd I would like to indeed have this optional and when enabled, it should fail secureboot.